pkg/clipboard/unclip.go

```diff
@@ -11,37 +13,68 @@ import (
 )
 
 // Clear will attempt to erase the clipboard.
-func Clear(ctx context.Context, checksum string, force bool) error {
-	if clipboard.Unsupported {
-		return ErrNotSupported
-	}
+func Clear(ctx context.Context, name string, checksum string, force bool) error {
+	clipboardClearCMD := os.Getenv("GOPASS_CLIPBOARD_CLEAR_CMD")
```

Comment:
Similar to `CopyTo`: I think this is getting too long to comprehend.

Reply:
Fixed in a0986a93